landing/introduction.mdx

```diff
@@ -7,52 +7,43 @@ iconType: "light"
 
 ## What is Particle Network?
 
-Particle Network allows developers to easily integrate social logins and account abstraction in decentralized applications, enhancing user experience and streamlining development.
+Particle Network radically simplifies user experience through Wallet Abstraction and Chain Abstraction. Through its Wallet-as-a-Service SDKs, developers can enable 2-click onboarding into smart accounts through social logins.
 
 ## Key Products
 
-Particle Network offers a suite of SDKs to simplify authentication:
-
-- **[Particle Connect](../api-reference/connect):** Connect to Web 2.0 accounts and mainstream wallets, enhancing accessibility through a unified interface.
-- **[Particle Auth](../api-reference/auth):** Onboard users with one-click authentication using popular platforms like Google, X, and many more.
-- **[Embedded Wallet](../api-reference/wallet):** Integrate a Web3 wallet directly into your dApp, allowing users to manage their assets without leaving your application.
-- **[Account Abstraction](../api-reference/aa):** Modularize your dApp into ERC-4337 AA, leveraging Particle’s Bundler, Paymaster, and compatibility with Smart Account implementations.
-- **[BTC Connect](../api-reference/btc):** Control a Smart Account on an EVM-compatible L2 using your Bitcoin wallet.
+Particle Network current product suite features numerous SDKs made to simplify authentication and onboarding:
 
 <CardGroup cols={2}>
+  <Card title="Particle Auth" icon="shield" href="../api-reference/auth">
```

Comment:
List of links were redundant, only using cards looks better.

Reply:
yep, left both to choose from